spec/oncocore.txt

```diff
@@ -189,37 +189,42 @@ Based on:		CodedLaboratoryObservation
 		                DataValue.Quantity.Units is UCUM#mm	
 
 
-EntryElement: 	TumorMargins
+EntryElement: 	TumorInvolvementAtSurgicalMargin
 Based on:		CodedLaboratoryObservation
-Concept:		MTH#C4086369
 Description:	"The edge or border of the tissue removed in cancer surgery. The margin is described as negative or clean when the pathologist finds no cancer cells at the edge of the tissue, suggesting that all of the cancer has been removed. The margin is described as positive or involved when the pathologist finds cancer cells at the edge of the tissue, suggesting that all of the cancer has not been removed."
-                DataValue from PositiveNegativeVS
-				Code is MTH#C4086369
+				Code is SCT#371488000 "Status of margin involvement by tumor (observable entity)"
+				DataValue from SurgicalMarginInvolvementVS
+// Specimen should be required!
 0..0            PanelMembers
-				Components.ObservationComponent
-					includes 0..1	TumorMarginDescription
-// Do we need to record the method? whether the result of imaging, or study of a removed tumor after lumpectomy?	
+0..0			Components
 
-		Element:		TumorMarginDescription
-		Based on:		CodedObservationComponent
-		Description:	"Description of the edge or border of tumor in situ by radiologist or of removed tumor by pathologist."
-						Code is TBD#TBD
-		                DataValue from TumorMarginDescriptionVS
 
 /*
+EntryElement:	TumorMarginDescription
+Based on:		CodedObservationComponent
+Description:	"Description of the edge or border of tumor in situ by radiologist or of removed tumor by pathologist."
+				Code is TBD#TBD
+		        DataValue from TumorMarginDescriptionVS
+
 Element:		TumorCellularity
 Concept:		MTH#C4055283
 Based on:		QuantitativeObservationComponent
 Description:	"Percentage of cells in a sample that are cancerous"
 				DataValue.Quantity.Units is UCUM#%
 
-
 Element:		PercentageOfInSituDisease
 Based on:		QuantitativeObservationComponent
 Concept:		TBD
 Description:	"The percentage of the cancer that is in situ, as opposed to invading other tissues."
 			    DataValue.Quantity.Units is UCUM#%
 
-*/
-
+EntryElement:	EarliestDateOfCancerCondition
+// TODO_discussion: Not convinced we need a separate element for this given the Onset property of Condition.
+Based on:		SimpleNonLaboratoryObservation
+Concept:		LNC#77975-1 "Earliest date of diagnosis"
```

Comment:
If the LOINC code is more generic, then could we just call this "EarliestDateOfDiagnosis" and then make this an attribute of CancerDisorder? This preserves context in a post-coordinated way and also allows for reusability to other diseases.